e2e/pages/adf/content-services.page.ts

```diff
@@ -608,9 +642,49 @@ export class ContentServicesPage {
         await BrowserActions.closeMenuAndDialogs();
         await BrowserActions.click(this.multiSelectToggle);
     }
+    async multiSelectToggleIsEnabled(): Promise<boolean> {
+        return this.multiSelectToggle.isEnabled();
+    }
+    async clickSelectAllCheckbox(): Promise<void> {
+        await BrowserActions.click(this.selectAllCheckbox);
+    }
 
     getRowByName(rowName): ElementFinder {
         return this.contentList.dataTable.getRow(this.columns.name, rowName);
     }
 
+    async selectFolder(folderName: string): Promise<void> {
+        const folderSelected: ElementFinder = element(by.css(`div[data-automation-id="${folderName}"] .adf-datatable-center-img-ie`));
+        await BrowserVisibility.waitUntilElementIsVisible(folderSelected);
+        await BrowserActions.click(folderSelected);
+    }
+
+    async selectFolderWithCommandKey(folderName: string): Promise<void> {
+        await browser.actions().sendKeys(protractor.Key.COMMAND).perform();
+        await this.selectRow(folderName);
+        await browser.actions().sendKeys(protractor.Key.NULL).perform();
+    }
+
+    async chooseSelectionMode(option: string): Promise<void> {
+        const dropdownPage = new DropdownPage(this.selectionModeDropdown);
+        await dropdownPage.clickDropdown();
+        await dropdownPage.selectOption(option);
```

Comment:
there is a method called clickDropdownWithOption which is wrapping this 2 methods into one. Can we use that one?

Reply:
done